assets/js/previews/grid-block.js

```diff
@@ -0,0 +1,388 @@
+export const gridBlockPreview = (
```

Comment:
This file is 5kb.  If only included with js that loads in the admin that might not be too bad because it'll be browser cached by the admin.  However keep in mind this is being embedded in every built block asset.  As we grow our stable of blocks that could potentially use this preview, that essentially duplicates the same 5kb across all those bundles (and starts to add up).  It may be something we want to load via path instead of embedded (so it's shared among all blocks using it).  Another advantage of path is _eventually_ it might be an asset that could be exposed for extensions to use.